common/client-core/src/client/packet_statistics_control.rs

```diff
@@ -479,11 +284,8 @@ impl PacketStatisticsControl {
                 },
                 _ = snapshot_interval.tick() => {
                     self.update_history();
-                    self.update_rates();
                 }
                 _ = report_interval.tick() => {
-                    self.report_rates();
```

Comment:
In the vpn cli client it's pretty useful to have the traffic data rates printed periodically, is there some way to get the equivalent functionality with this new setup?

Reply:
Yes, we can even put the packet rate calculations back, I wasn't sure how this was gonna end up eventually, will put something back